integration_test/third_party_apps_data/applications/varnish/metadata.yaml

```diff
@@ -1,5 +1,6 @@
 short_name: varnish
 long_name: Varnish HTTP Cache
+supported_app_version: ["6.0+", "7.0+"]
```

Comment:
@JonathanWamsley Could you confirm that this is the correct supported version for Varnish?

Reply:
We don't use + but rather x to indicate all the versions in the series. So `["6.x", "7.0.x"]` like in the [scope](https://docs.google.com/document/d/1WyQ42YPsRodVKbk1_kvEBTZT7RDvzhXzHtlgEWOxaE8/) doc. This is is what other receivers like [wildfly](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/third_party_apps_data/applications/wildfly/metadata.yaml#L32) does too.

Reply:
Done.